stand-alone/package.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "@fandom/rv-jwplayer",
+	"version": "2.1.5",
```

Comment:
The version and description should probably be changes to reflect this app instead of the library.

Reply:
Fixed in 11b086d1be3a655e8ad24d9eceaff73ae0f51813